app/src/main/java/com/hedvig/app/feature/embark/passages/previousinsurer/retrieveprice/RetrievePriceActivity.kt

```diff
@@ -0,0 +1,110 @@
+package com.hedvig.app.feature.embark.passages.previousinsurer.retrieveprice
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.material.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.res.stringResource
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.hedvig.app.R
+import com.hedvig.app.feature.settings.Market
+import com.hedvig.app.ui.compose.composables.CenteredProgressIndicator
+import com.hedvig.app.ui.compose.composables.ErrorDialog
+import com.hedvig.app.ui.compose.composables.FadeWhen
+import com.hedvig.app.ui.compose.composables.appbar.TopAppBarWithBack
+import com.hedvig.app.ui.compose.theme.HedvigTheme
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class RetrievePriceInfoActivity : ComponentActivity() {
```

Comment:
Is there a particular reason this does not inherit from BaseActivity? I don't think I am familiar with what ComponentActivity does differently from AppCompatActivity